Runtime error at start up after update to 0.6.8

I have recently updated Chaind from 0.6.5 to 0.6.8. 

The service panics and exits with the following while starting up.

```
{"level":"info","version":"0.6.8","time":"2022-06-13T15:24:07Z","message":"Starting chaind"}
{"level":"debug","time":"2022-06-13T15:24:07Z","message":"No metrics service supplied; monitor not starting"}
{"level":"debug","time":"2022-06-13T15:24:07Z","message":"no metrics will be generated for this module"}
{"level":"info","profile_address":":1234","time":"2022-06-13T15:24:07Z","message":"Starting profile server"}
{"level":"info","service":"synccommittees","impl":"standard","period":491,"time":"2022-06-13T15:24:07Z","message":"Catching up from period"}
{"level":"info","service":"synccommittees","impl":"standard","time":"2022-06-13T15:24:07Z","message":"Caught up"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x9a0eb7]

goroutine 1 [running]:
github.com/wealdtech/chaind/services/blocks/standard.monitorLatestBlock(...)
	/app/services/blocks/standard/metrics.go:74
github.com/wealdtech/chaind/services/blocks/standard.New({0xdd2fb0, 0xc000228b40}, {0xc0000edc40, 0x0, 0x0})
	/app/services/blocks/standard/service.go:135 +0x917
main.startBlocks({0xdd2fb0, 0xc000228b40}, {0xdc95c0, 0xc00024e3c0}, {0xdd3250, 0xc00022e298}, {0xde4f28, 0xc0002c53c0}, {0xdbf640, 0x134a930})
	/app/main.go:462 +0x3f6
main.startServices({0xdd2fb0, 0xc000228b40}, {0xdbf640, 0x134a930})
	/app/main.go:337 +0x8d3
main.main2()
	/app/main.go:116 +0x52a
main.main()
	/app/main.go:64 +0x19

```

`latestBlock` var seems to be uninitialised [here](https://github.com/wealdtech/chaind/blob/master/services/blocks/standard/metrics.go#L74) (the method is called [here](https://github.com/wealdtech/chaind/blob/master/services/blocks/standard/service.go#L135))

--- a/services/blocks/standard/metrics.go
+++ b/services/blocks/standard/metrics.go
@@ -71,7 +71,9 @@ func registerPrometheusMetrics(ctx context.Context) error {
 // called directly, as it is called as part ofr monitorBlockProcessed.
 func monitorLatestBlock(slot phase0.Slot) {
 	highestSlot = slot
-	latestBlock.Set(float64(slot))
+	if latestBlock != nil {
+		latestBlock.Set(float64(slot))
+	}
 }
 
 func monitorBlockProcessed(slot phase0.Slot) {

--- a/services/finalizer/standard/metrics.go
+++ b/services/finalizer/standard/metrics.go
@@ -70,7 +70,9 @@ func registerPrometheusMetrics(ctx context.Context) error {
 // called directly, as it is called as part ofr monitorEpochProcessed.
 func monitorLatestEpoch(epoch phase0.Epoch) {
 	highestEpoch = epoch
-	latestEpoch.Set(float64(epoch))
+	if latestEpoch != nil {
+		latestEpoch.Set(float64(epoch))
+	}
 }
 
 func monitorEpochProcessed(epoch phase0.Epoch) {

--- a/services/summarizer/standard/metrics.go
+++ b/services/summarizer/standard/metrics.go
@@ -70,7 +70,9 @@ func registerPrometheusMetrics(ctx context.Context) error {
 // called directly, as it is called as part ofr monitorEpochProcessed.
 func monitorLatestEpoch(epoch phase0.Epoch) {
 	highestEpoch = epoch
-	latestEpoch.Set(float64(epoch))
+	if latestEpoch != nil {
+		latestEpoch.Set(float64(epoch))
+	}
 }
 
 func monitorEpochProcessed(epoch phase0.Epoch) {

